site_root/input/resources/examples/DocumentReference-example-document-reference-id.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resourceType": "DocumentReference",
+  "id": "example-document-reference-id",
+  "meta": {
+    "profile": [
+      "http://fhir.ncpi-project-forge.io/StructureDefinition/ncpi-drs-document-reference"
+    ]
+  },
+  "status": "current",
+  "identifier": [
+    {
+      "system": "urn:ncpi:unique-string",
+      "value": "example-document-reference-id"
+    }
+  ],
+  "content": [
+    {
+      "attachment": {
+        "contentType": "application/json",
```

Comment:
Confirmed Ontoserver hasn't yet added the mime type CodeSystem. We may want to either add the CodeSystem or remove `"contentType"` for now.

Reply:
resolved.